jupyter/pytorch/ubi9-python-3.9/Dockerfile

```diff
@@ -14,6 +14,9 @@ LABEL name="odh-notebook-jupyter-pytorch-ubi9-python-3.9" \
 # Install Python packages and Jupyterlab extensions from Pipfile.lock
 COPY Pipfile.lock ./
 
+# Switch to root user
+USER root
```

Comment:
any specific reason for switching to root here?

Reply:
There was a permission issue when attempting to write the pipfile.lock file to the container image. [Error:-Failed to write lockfile to container image: [Errno 13] Permission denied: 'Pipfile.lock']
hence above changes were done.

Reply:
lets debug that instead of making root context here, use podman.. or check if you are facing any selinux related issue etc..

Reply:
@mkumatag  @npanpaliya 
I haven't found any other error in the logs besides this 'permission denied'( [Error:-Failed to write lockfile to container image: [Errno 13] Permission denied: 'Pipfile.lock']) while building the image. 
Instead of using the root user, can we use then:-
**COPY --chown=1001:0 Pipfile.lock ./**
Could you please guide me further on that

Reply:
my point is if this docker file is built on x86 then same file should on ppc64le as well, I don't see a point in modifying just for power!

Reply:
is this because of the different base image? 🤔 if yes - lets fix the base image accordingly..

Reply:
@mkumatag  @npanpaliya 

1. After updating the notebook repository to the latest tag v1.20.0, I have successfully built the PyTorch image.
No root user changes are required inside this  Dockerfile. All packages installed successfully into the container.
**Snap:-**
<img width="697" alt="powercommand_snap" src="https://github.com/user-attachments/assets/0185ffef-4203-4001-a68b-3024e2a2bba2">

2. Additionally, I tested the base image building on an x86 VM. It was observed that the build encounters the same error "failed to write lockfile to container image." However, this error does not impact any functionality and can be safely ignored.

**Snap:-**
<img width="739" alt="x86_baseimage" src="https://github.com/user-attachments/assets/b9f88013-91f3-49a8-b81f-75623686ba38">

3. I need to close this PR because the base branch has been updated with the latest changes. I will create a new PR for this pytorch and datascience image builds changes."